pybamm/input/parameters/lithium_ion/cells/LGM50_ORegan2021/parameters.csv

```diff
@@ -0,0 +1,34 @@
+Name [units],Value,Reference,Notes
+# Empty rows and rows starting with ‘#’ will be ignored,,,
+,,,
+# Macroscale geometry,,,
+Negative current collector thickness [m],12E-6,Chen 2020,
+Negative electrode thickness [m],85.2E-6,Chen 2020,
+Separator thickness [m],12E-6,Chen 2020,
+Positive electrode thickness [m],75.6E-6,Chen 2020,
+Positive current collector thickness [m],16E-6,Chen 2020,
+Electrode height [m],6.5E-2,Chen 2020,Not needed for 1D
```

Comment:
I think we also have the comment "Not needed for 1D" in the Chen2020 set, but the height and width are used to calculate the electrode cross-sectional area, which is used in 1D. Maybe we should update the comments to say the width is the "unwound" length of the jelly roll?